yarn.lock

```diff
@@ -5832,6 +5825,17 @@ cosmiconfig@^6.0.0:
     path-type "^4.0.0"
     yaml "^1.7.2"
 
+cosmiconfig@^7.0.0:
+  version "7.0.0"
+  resolved "https://artifactory.twitter.biz:443/api/npm/js-virtual/cosmiconfig/-/cosmiconfig-7.0.0.tgz#ef9b44d773959cae63ddecd122de23853b60f8d3"
```

Comment:
Looks like you've got a global `~/.npmrc` that's getting inherited here :wink:. If you edit and comment out the registry lines in, then re-running `yarn` should fix this up – potentially have to revert the file first.

Reply:
😂  oops! fixed